src/ripple/app/reporting/ETLSource.h

```diff
@@ -374,7 +374,7 @@ class ETLLoadBalancer
     /// @param in ETLSource in question
```

Comment:
Update the documentation for this function which currently only describes the transaction stream.